src/test/java/hudson/remoting/TrafficAnalyzer.java

```diff
@@ -21,23 +21,24 @@ public static void main(String[] args) throws Exception {
         File f = new File("/home/kohsuke/ws/hudson/investigations/javafx-windows-hang/out.log");
         DataInputStream fin = new DataInputStream(new FileInputStream(f));
         fin.readFully(new byte[4]); // skip preamble
-        ObjectInputStream ois = new ObjectInputStream(fin);
-        for (int n=0; ; n++) {
-            Command o = (Command)ois.readObject();
-            System.out.println("#"+n+" : "+o);
-            if (o instanceof RPCRequest) {
-                RPCRequest request = (RPCRequest) o;
-                System.out.print("  (");
-                boolean first=true;
-                for (Object argument : request.getArguments()) {
-                    if(first)   first=false;
-                    else        System.out.print(",");
-                    System.out.print(argument);
+        try (ObjectInputStream ois = new ObjectInputStream(fin)) {
+            for (int n=0; ; n++) {
+                Command o = (Command)ois.readObject();
+                System.out.println("#"+n+" : "+o);
+                if (o instanceof RPCRequest) {
+                    RPCRequest request = (RPCRequest) o;
+                    System.out.print("  (");
+                    boolean first=true;
+                    for (Object argument : request.getArguments()) {
+                        if(first)   first=false;
+                        else        System.out.print(",");
+                        System.out.print(argument);
+                    }
+                    System.out.println(")");
                 }
-                System.out.println(")");
+                if (o.createdAt!=null)
+                    o.createdAt.printStackTrace(System.out);
             }
-            if (o.createdAt!=null)
-                o.createdAt.printStackTrace(System.out);
         }
     }
 }
```

Comment:
If the earlier comment about including the input in a try with resources is correct and useful, then something like this will be needed
```suggestion
    }
}
```